src/components/shared/thread/ActivityGraphFills.js

```diff
@@ -679,8 +673,7 @@ export class ActivityFillGraphQuerier {
    * been applied.
    */
   _getSmoothedContributionFromSampleToPixel(
-    xPixel: number,
-    xPixelsPerMs: number,
+    mouseTime: Milliseconds,
```

Comment:
So I think it would be better to keep passing an integer `xPixel` argument to this function. We can compute the kernelRangeStartTime from the kernelPos at the end of this function, just before calling `_accumulateInBuffer`.

Reply:
Oh, right. When I was looking at this, I was thinking that passing the `time` directly would reduce some operations. But I changed it since misaligns with the rendering grid. Thanks for pointing that out. Would you mind taking a look at the last commit again?